frontend/src/container/NotificationWall.jsx

```diff
@@ -119,73 +115,74 @@ export class NotificationWall extends React.Component {
     const i18nOpts = {
       user: `<span title='${escapedUser}'>${escapedUser}</span>`,
       author: `<span title='${escapedAuthor}'>${escapedAuthor}</span>`,
-      content: `<span title='${escapedContentLabel}' class='contentTitle__highlight'>${escapedContentLabel}</span>`,
+      content: `<span title='${escapedContentLabel}' class=${notification.numberOfContents > 1
+        ? ''
+        : 'contentTitle__highlight'
+      }>${escapedContentLabel}</span>`,
       interpolation: { escapeValue: false }
     }
 
+    if (notification.numberOfWorkspaces > 1) {
+      i18nOpts.workspace = `<span title='${notification.numberOfWorkspaces}'>${
+        props.t('in {{numberOfWorkspaces}} spaces', { numberOfWorkspaces: notification.numberOfWorkspaces })
+      }</span>`
+    }
+
     const isPublication = notification.content && notification.content.contentNamespace === CONTENT_NAMESPACE.PUBLICATION
 
     const contentUrl = (
       notification.content
         ? (
           isPublication
             ? PAGE.WORKSPACE.PUBLICATION(notification.workspace.id, notification.content.id)
-            : PAGE.WORKSPACE.CONTENT(notification.workspace.id, notification.content.type, notification.content.id)
+            : PAGE.CONTENT(notification.content.id)
         )
         : ''
     )
 
     if (entityType === TLM_ENTITY.CONTENT) {
-      const publicationIcon = isPublication ? 'fas fa-stream+' : ''
-
       switch (eventType) {
         case TLM_EVENT.CREATED: {
           if (contentType === TLM_SUB.COMMENT) {
             return {
-              icon: 'far fa-comments',
               title: props.t('Comment_noun'),
-              text: props.t('{{author}} commented on {{content}}', i18nOpts),
+              text: props.t('{{author}} commented on {{content}} {{workspace}}', i18nOpts),
```

Comment:
This means that the string ends with a space if workspace is empty. I think this translation key could be quite confusing to a non-dev translator, I think I'd prefer keeping it the way it was and add the contents of workspace in the content key. Possibly renaming the key so it is explicit that {{content}} is actually the name of a content + a chunk of sentence related to the workspace.
Or at least changing {{{workspace}} to be something that explicitly says that it actually contains a chunk of sentence.

No big opinion though.

(same applies for the other occurrences below)

Reply:
If you are ok with it, I can make an issue to address this problem

Reply:
Our organization by sprints prompts me to tell you ok, and to close my eyes at the fact that this technical dept issue will probably collect some dust.